src/client/components/AddMeals.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+
+function AddMeals() {
+  const [title, setTitle] = useState("");
```

Comment:
I would prefer to separate these input fields into their own components. I agree its not terribly critical here. But it allows you to make individual changes to each component and be confident that the changes will not interfere with the rest.